Epam.TestAutomation.Tests.BDD/Features/Epam_DDT_SelectedLocations.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Epam_DDT_SelectedLocations
+As an automation engineer
+I want to test Epam site for career opportunities
+by selecting different locations
+and verify that returned results match
+
+    @automation @DDT
+    Scenario Outline: Check that search results contain selected locations
+        Given I navigate to Epam site
+        Given I accept all cookies on Epam site
+        And navigate to Careers page
+        When I click on Location drop-down I select "<country>" and "<city>"
+        And click Find button
```

Comment:
Please review steps naming. What – Where, Ex: I click to search field -> I click on Search field on Search Panel on Epam Careers page